cmake/deps.cmake

```diff
@@ -26,13 +26,13 @@ set(GRPC_GIT_TAG "883e5f76976b86afee87415dc67bde58d9b295a4") # v1.59.2
 set(GRPC_VERSION "1.59.2")
 
 set(GTEST_GIT_URL "https://github.com/google/googletest.git")
-set(GTEST_GIT_TAG "e2239ee6043f73722e7aa812a459f54a28552929") # v1.11.0
+set(GTEST_GIT_TAG "e2239ee6043f73722e7aa812a459f54a28552929") # release-1.11.0
 
 set(JSON_GIT_URL "https://github.com/nlohmann/json.git")
-set(JSON_GIT_TAG "760304635dc74a5bf77903ad92446a6febb85acf") # tags/v3.10.5^2~8
+set(JSON_GIT_TAG "760304635dc74a5bf77903ad92446a6febb85acf") # v3.10.1-30-g76030463
 
 set(PROTOBUF_GIT_URL "https://github.com/google/protobuf.git")
-set(PROTOBUF_GIT_TAG "6b5d8db01fe47478e8d400f550e797e6230d464e") # v25.0
+set(PROTOBUF_GIT_TAG "6b5d8db01fe47478e8d400f550e797e6230d464e") # v3.25.0
```

Comment:
This should just be `25.0` and not `3.25.0`. 
IIRC, protobuf has moved on to a simpler numbering scheme and instead of 3.25.0, and 4.25.0, they just started doing a 25.0 as the release/tag